src/test/java/seedu/duke/command/SeeAllIncomesCommandTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.financial.Expense;
+import seedu.duke.financial.FinancialList;
+import seedu.duke.financial.Income;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
maybe should try below instead of `import static org.junit.jupiter.api.Assertions.*;`
```
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Assertions.assertTrue;
```